docs/sources/flow/setup/install/windows.md

```diff
@@ -9,29 +9,31 @@ aliases:
 
 # Install Grafana Agent in flow mode on Windows
 
-You can install Grafana Agent in flow mode on Windows with the standard graphical installer, or as a silent install.
+You can install Grafana Agent in flow mode on Windows as a standard install, or as a silent install.
```

Comment:
Is "standard install" well known to mean graphical? If so, this is fine.

Reply:
I've reverted this in the next commit.